packages/elements/src/clock/__test__/clock.interactive.test.js

```diff
@@ -18,17 +22,21 @@ describe('clock/Interactive', () => {
     }
 
     const InputKey = {
-      arrowUp: {
+      ArrowLeft: {
+        ieKey: 'Left',
```

Comment:
When do you use `ieKey` ?

Reply:
Updated to use iekey. https://github.com/Refinitiv/refinitiv-ui/pull/308/commits/174dfccb62ceaa7662e9633281a50888009d85cf